docs/examples/module_step2_optional.py

```diff
@@ -48,6 +48,10 @@ def reconfigure(self, config: ComponentConfig, dependencies: Mapping[ResourceNam
             multiplier = 1.0
         self.multiplier = multiplier
 
+    def close(self):
+        # This is a completely optional function to include. This will be called when the resource is closed.
+        print(f"{self.name} is closed.")
```

Comment:
we should use the logger instead - print statements don't get propagated to the RDK